packages/mail/src/classes/mail-service.js

```
@@ -110,6 +112,17 @@ class MailService {
     });
   }
 
+  /**
+   * Set default data to for all requests. Can be overwritten by incoming data on send
+   */
+  setDefaultData(data) {
+    if (Object.prototype.toString.call(data) !== "[object Object]") {
```

What is this doing?

---

Checking data is an object.

---

Prefer to throw an exception if it's not an object instead of just returning.